Makefile

```diff
@@ -277,6 +277,7 @@ test: ginkgo-set tidy-vendor
 
 test-verbose: ginkgo-set tidy-vendor
 	@echo TAGS=$(GO_BUILD_TAGS)
+	@echo GOENV=$(GOENV)
```

Comment:
do we need echo this?

Reply:
the information echoed in this line is helpful in checking errors we get from `go build`, if any.